Makefile

```
@@ -480,13 +485,24 @@ install-coqutil:
 	$(MAKE) --no-print-directory -C $(COQUTIL_FOLDER) install
 
 bedrock2: coqutil
-	$(MAKE) --no-print-directory -C $(BEDROCK2_FOLDER) noex
+	$(MAKE) --no-print-directory -C $(BEDROCK2_ROOT_FOLDER) bedrock2_noex
 
 clean-bedrock2:
-	$(MAKE) --no-print-directory -C $(BEDROCK2_FOLDER) clean
+	$(MAKE) --no-print-directory -C $(BEDROCK2_ROOT_FOLDER) clean_bedrock2
 
 install-bedrock2:
-	$(MAKE) --no-print-directory -C $(BEDROCK2_FOLDER) install
+	$(MAKE) --no-print-directory -C $(BEDROCK2_ROOT_FOLDER) install_bedrock2
+
+bedrock2-compiler: bedrock2
+	# We have to build the compiler *including* examples to get compilerExamples/MMIO.v
```

Is it possible that `make` doesn't like recipes that start with `#`?  What happens if you do
```suggestion
	@# We have to build the compiler *including* examples to get compilerExamples/MMIO.v
```
instead?

---

Adjusted to try that. The "no rule" error is especially weird because this does build locally with `make`.

---

Same error. Fascinating.

---

Strange.  Does coq-record-update build fine for you locally?  Maybe it makes sense to first fix the coq-record-update failure and then look into this failure after?